### ISSUE TITLE
Where are the Delphi Pascal communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/delphi#232]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Delphi Pascal supported by one or more large organizations?
* Does Delphi Pascal have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Delphi Pascal? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Delphi Pascal? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Delphi Pascal? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Delphi Pascal?
* Are there popular newsletters for Delphi Pascal?
* Is Delphi Pascal taught at programming bootcamps? (If so, what are some examples?)
* Is Delphi Pascal taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Delphi Pascal?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by rpottsoh** [comment]
Thanks @jimmckeeth for helping out and posting this information here.

@Kytrinyx it appears to me that the information that @jimmckeeth has provided touches on most, perhaps all of the points you've listed.  It isn't clear to me what should be done with this issue.  My presumption is to close it.  Which I think I will do, it can always be reopened if further action is needed.

**Comment by kytrinyx** [comment]
@jimmckeeth @rpottsoh Yes, this is great, thank you. Feel free to close (as you've done), once you feel like the topic is exhausted. I will probably go through and close everything that has received a response in a week or two, as I'm really just collating information for future use.
